pyuvdata/uvdata/uvh5.py

```diff
@@ -287,13 +294,13 @@ def _read_header(
             self.flex_spw_id_array = header["flex_spw_id_array"][:]
         if "flex_spw_polarization_array" in header:
             self.flex_spw_polarization_array = header["flex_spw_polarization_array"][:]
+        read_phase_center_catalog = False
+        if "phase_center_catalog" in header:
+            read_phase_center_catalog = True
 
         # Here is where we start handing phase center information.  If we have a
         # multi phase center dataset, we need to get different header items
-        if "phase_center_catalog" in header:
-            self._set_phased()
-            self._set_multi_phase_center(preserve_phase_center_info=False)
-
+        if read_phase_center_catalog:
```

Comment:
Do we need to have this extra variable defined here?
why not just check if `phase_ceneter_catalog` in header on line 303?

Reply:
good point, I'll take it out.